src/screens/FavoritesScreen.js

```diff
@@ -0,0 +1,35 @@
+import React, {Component} from 'react';
+import {Text} from 'react-native';
+import {Icon} from 'react-native-elements';
+
+
+import {CardSection, Card} from '../components/common/index';
+import MapScreen from "../components/SearchMap";
+
+class FavoritesScreen extends Component {
+    static navigationOptions = (props) => {
+        const {navigation: {navigate}} = props;
+
+        return {
+            title: 'Favorites',
+            headerTitle: 'Your Favorite Campsites',
+            headerLeft: null,
+            tabBarIcon: ({focused, tintColor }) => (<Icon type='ionicon' name={focused ? 'ios-heart' : 'ios-heart-outline'} size={25} color={tintColor} />)
```

Comment:
Really great use of `react-navigation` together with `react-native-elements`. Things like `focused` and `tintColor` come for free, and so we can easily do things like toggle color or icon based on state.